pkg/vere/io/ames.c

```diff
@@ -2458,6 +2461,11 @@ u3_ames_io_init(u3_pier* pir_u)
   sam_u->fig_u.see_o = c3y;
   sam_u->fig_u.fit_o = c3n;
 
+  //  enable forwarding on galaxies only
+  u3_noun who = u3i_chubs(2, sam_u->pir_u->who_d);
+  u3_noun rac = u3do("clan:title", u3k(who));
```

Comment:
`who` is leaked here.  Probably easiest to just remove the `u3k` to transfer it into `u3do`